pkg/conthandler/container_aggregator.go

```diff
@@ -26,7 +26,7 @@ func (aggregator *Aggregator) collectDataFromContainerAccumulator(errChan chan e
 	for {
 		newEvent := <-aggregator.aggregationDataChan
 		if newEvent.GetEventCMD() == accumulator.DropEventOccurred {
-			// aggregator.StopAggregate()
```

Comment:
I removed it by purpose, the management decided to continue listen on container when we got dropped events.